docs/guide/service/annotations.md

```diff
@@ -7,6 +7,7 @@
         - stringList: `"s1,s2,s3"`
         - stringMap: `"k1=v1,k2=v2"`
         - json: `"{ \"key\": \"value\" }"`
+    - The controller may not update the configuration of an *existing* load balancer if an annotation is added/updated. New load balancers will be have an appropriate configuration. 
```

Comment:
Barring some limitations, controller does update the existing load balancer. There are some issues with the kube-controller-manager provisioned load balancer, but this controller should be able to update most of the configuration.

Limitations:
- modifying/adding the annotation `service.beta.kubernetes.io/aws-load-balancer-type` after service creation is not recommended as there is a potential for leaking the AWS LB resource in case both kcm and this controller act on the service
- if user modifies the lb scheme, controller deletes and creates a new lb with the new scheme
- updating subnets may not work - this is due to any temporary limitation put in by the AWS